hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/domain/ContractResultServiceImpl.java

```diff
@@ -153,6 +220,30 @@ private void processContractLogs(ContractFunctionResult functionResult, Contract
         }
     }
 
+    private void processContractStateChanges(List<ContractStateChange> stateChangeList, ContractResult contractResult) {
+        for (ContractStateChange stateChange : stateChangeList) {
+            var contractId = lookup(contractResult.getContractId(), stateChange.getContractId());
+            for (var storageChange : stateChange.getStorageChangesList()) {
+                com.hedera.mirror.common.domain.contract.ContractStateChange contractStateChange =
```

Comment:
Use var to partially hide the ugliness of the similar type.
```suggestion
                var contractStateChange = new com.hedera.mirror.common.domain.contract.ContractStateChange();
```

Reply:
Thanks I have switched it to a var.